From: =?utf-8?b?Ik1hcmNvIFRyZXZpc2FuIChUcmV2acOxbyki?= <mail@3v1n0.net>
Date: Fri, 28 Aug 2020 13:59:56 +0200
Subject: meson_options: Enable systemd-journal by default

This was already the case in autotools, as it used to be a feature that was
set to auto and disabled only in the case there was no libsystemd.

Now, being systemd a required gdm component, there's no point to use it as
a feature, but still give the packagers the ability to toggle it.

Origin: https://gitlab.gnome.org/GNOME/gdm/-/merge_requests/107
Forwarded: yes
---
 meson_options.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson_options.txt b/meson_options.txt
index 040fe38..9a9f76e 100644
--- a/meson_options.txt
+++ b/meson_options.txt
@@ -27,7 +27,7 @@ option('selinux', type: 'feature', value: 'auto', description: 'Add SELinux supp
 option('solaris', type: 'boolean', value: false, description: 'Build for Solaris')
 option('split-authentication', type: 'boolean', value: true, description: 'Enable multiple simultaneous PAM conversations during login.')
 option('sysconfsubdir', type: 'string', value: 'gdm', description: 'Directory name used under sysconfdir.')
-option('systemd-journal', type: 'boolean', value: false, description: 'Add journald support.')
+option('systemd-journal', type: 'boolean', value: true, description: 'Add journald support.')
 option('systemdsystemunitdir', type: 'string', value: '', description: 'Directory for systemd service files.')
 option('systemduserunitdir', type: 'string', value: '', description: 'Directory for systemd user service files.')
 option('tcp-wrappers', type: 'boolean', value: false, description: 'Use TCP wrappers.')
